### ISSUE TITLE
Čech complex in python

### ISSUE THREAD

**Issue by chujie** [GUDHI/gudhi-devel#394]
Čech complex is not available in python interface. Is there any way to use it in python code?

**Comment by mglisse** [comment]
Indeed, it isn't available yet. Adding such an interface would be relatively simple, but it hasn't been a priority, since in most cases it is preferable to use the alpha-complex or the Rips complex. Do you have a specific use case where you really need the Čech complex?

**Comment by chujie** [comment]
I am studying the sensor coverage problem, where Čech complex should be the best fit.
